### PR TITLE
Ncgames 10

### PR DESCRIPTION
**Pull request by FisherTom**
add category, sortby and order queries to /api/reviews and tests

### CODE REVIEW

**Review by NRMSMITH — CHANGES_REQUESTED** [review]
-> need to get rid of .skip before we can do anything

File: __tests__/app.test.js [hunk 1]
```diff
@@ -51,14 +51,80 @@ describe("GET requests", () => {
...
+          expect(reviews.length).toBeGreaterThan(0);
+          reviews.forEach((review) => {
+            expect(review.category).toBe("dexterity");
+          });
+        });
+    });
+    test("should accept a sort_by query to sort results by selected collumn", () => {
+      return request(app)
+        .get("/api/reviews?sort_by=votes")
+        .then((response) => {
```
Comment (NRMSMITH) [inline __tests__/app.test.js:77]: expect(200) as well

File: __tests__/app.test.js [hunk 2]
```diff
@@ -51,14 +51,80 @@ describe("GET requests", () => {
...
+        .get("/api/reviews?category=social deduction&sort_by=votes&order=ASC")
+        .then((response) => {
+          const reviews = response.body.reviews;
+          expect(reviews.length).toBeGreaterThan(0);
+          expect(reviews).toBeSortedBy("votes", { ascending: true });
+          reviews.forEach((review) => {
+            expect(review.category).toBe("social deduction");
+          });
+        });
+    });
```
Comment (NRMSMITH) [inline __tests__/app.test.js:103]: add another 200 test for a valid category that doesn't have any reviews - should return an empty array

**Review by NRMSMITH — CHANGES_REQUESTED** [review]
Add one more test and make sure you're expecting a status code for each test

**Review by NRMSMITH — APPROVED** [review]
🚀
